features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Permission.js

```diff
@@ -430,32 +454,32 @@ class Permission extends Component {
                                         <Row>
                                             <Col span={4} style={{margin: "10px"}}>
                                                 <Input name="roleField" placeholder="role" value={this.state.roleField}
```

Comment:
Please remove antd and use material-ui related components for checkbox, and other stuff